File: d2024_02_12_Problema_00169/modelo5.rs
impl Solution {
    pub fn majority_element(nums: Vec<i32>) -> i32 {
        let mut valores = std::collections::HashMap::new ();
        let limite = nums.len () / 2;
        println! ("Limite: {limite}");
        for v in nums {
            let a = valores.entry (v).or_insert (0 as usize);
            *a += 1;
            if *a > limite {
                return v;
            }
        }
        0 // Para ter algo aqui
    }
}

File: d2021_08_02/modelo5.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let t = nums.length
    for (let i = 0; i < t; i += 1) {
        for (let j = i + 1; j < t; j += 1) {
            if (nums [i] + nums [j] == target) {
                return [i, j];
            }
        }
    }
    return [0, 0]; // Só para ter algo aqui
};
